deploy/src/main/deploy/helm/templates/_helpers.tpl

```diff
@@ -51,6 +51,7 @@ The scope passed in is expected to be a dict with keys
 */}}
 {{- define "hono.metadata" -}}
 name: {{ .dot.Release.Name }}-{{ .name }}
+namespace: {{ .dot.Release.Namespace }}
```

Comment:
not sure if `.Release.Namespace` will always have a value, i.e. even if you do not specify the `--namespace` argument.
It might be safer to wrap the whole property in that case, so that it is omitted if there is no namespace value:
```
{{- with .dot.Release.Namespace }}
namespace: {{ . }}
{{. end }}
```
WDYT?

Reply:
Yes, it always has a value: https://github.com/helm/helm/blob/master/cmd/helm/template.go#L136